compiler/src/dotty/tools/dotc/semanticdb/PPrint.scala

```diff
@@ -0,0 +1,252 @@
+package dotty.tools.dotc.semanticdb
+
+import dotty.tools.dotc.{semanticdb => s}
+
+import scala.collection.mutable
+import dotty.tools.dotc.semanticdb.Scala3.{_, given}
+import SymbolInformation.Kind._
+
+class SymbolInfomationPrinter (symtab: PrinterSymtab):
```

Comment:
We discussed with @tanishiking that the alternative to having a duplicate semanticdb printer would be to run `metap` manually. We could use coursier boostrap and add a minimal jar, which can be run and printed to file. What do you think @bishabosha ?

Reply:
I think another alternative to try is you can load metap with reflection maybe, but if this works then fine

Reply:
I have found however that our metap prints more information, for example in the occurrences we print the substring from the source file that matches the range, which metap does not do